### ISSUE TITLE
Question about selective search cached boxes in training and validation

### ISSUE THREAD

**Issue by CQIITLAB** [amirbar/DETReg#10]
Why are there some '.npy' files for the Imagnet validation set in 'ss_box_cache.tar.gz', for example ILSVRC2012_val_00000006.npy. Are training sets and validation sets used for pretraining?
